src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const tabHead = document.querySelector('thead');
+const tabBody = document.querySelector('tbody');
+
+tabHead.addEventListener('click', (events) => {
+  const cellInd = events.target.cellIndex;
+  const copyTabBodyCh = [...tabBody.children];
+
+  const sort = copyTabBodyCh.sort((a, b) => {
```

Comment:
Don't use the method's name as a name for your variable. 
Make the name more descriptive